lib/config.js

```diff
@@ -70,7 +70,11 @@ exports.load = function(fileName, currentEnv) {
   } else if(config['default']) {
     out.setCurrent(config['default']);
   } else if(config.env) {
-    out.setCurrent(config.env);
+    if (config.env.ENV) {
```

Comment:
Sure thing! Do you prefer it to be snake case `default_env` or camel cased `defaultEnv`? I noticed that one of the other options `multipleStatements` was camel cased.